printer_mods/Ferdi360/2.4_Bedfan_Backplates/.metadata.yml

```
@@ -0,0 +1,27 @@
+---
+# The title of your mod. Choose a short and descriptive name
+title: 2.4 Bedfan Backplates
+# A short description of your mod. Try to describe your mod with 1-2 short sentences
+description: Backplates for your Voron 2.4 that block hot air out of your bedfans from blowing onto your ACM panels to (slightly) improve airflow and temps. Also a nice way to showoff Voron branding and the serial of your printer.
+# Mod version: Set this to 1 when you are submitting a new mod. When you make changes to your mod, you may increment this number
+mod_version: 1
+# You may add your discord username here so people can contact you about your mod
+discord_username: ferdimac
+# The CAD files for your mod. Please provide them as a list.
+# Note that each mod must include at least one CAD file.
+cad:
+- CAD/2.4_Backplates_v1.f3d
+- CAD/2.4_Backplates_v1.step
+# The image files provided with your mod. These may be instructions you use in your README.md file, or other additional pictures.
+# **Note that the first image specified here will be used as cover image on mods.vorondesign.com**
+# All paths are relative to this .metadata.yml file.
+images:
+- Images/2.4_Backplates.png
```

It looks like this got changed to lowercase `.png` on your last commit, but the file is actually `.PNG` 

```suggestion
- Images/2.4_Backplates.PNG
```